test_helpers.go

```diff
@@ -64,3 +64,16 @@ func (v *ValStringStub) Includes(id NodeID) bool {
 func (v *ValStringStub) Len() int {
 	return len(*v)
 }
+
+func (v *ValStringStub) VotingPower() map[NodeID]uint64 {
+	return CreateEqualWeightValidatorsMap(*v)
+}
+
+// CreateEqualWeightValidatorsMap is a helper function which creates map with same weight for every validator id in the provided slice
+func CreateEqualWeightValidatorsMap(validatorIds []NodeID) map[NodeID]uint64 {
+	weightedValidators := make(map[NodeID]uint64, len(validatorIds))
+	for _, validatorId := range validatorIds {
+		weightedValidators[validatorId] = 1
```

Comment:
This change confuses me because this struct is using in TestRoundChange_PropertyMajorityOfVotingPowerAggreement. It seems we need correct voting power there.

Reply:
I don't get it, we don't use this function (neither `ValStringStub`) in this particular test, but instead setting voting metadata explicitly to the `currentState` reference:
https://github.com/0xPolygon/pbft-consensus/pull/69/files#diff-b2f021d366e25d5b9980cd37386d09c568352c340e3bfebaf58034bb1a7000f4R819.

Seems like there is no problem to me. Although agree that property tests could be even more consolidated.